test/TestAssets/CUITTestProject/CUITTestProject.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Package dependency versions -->
+  <PropertyGroup>
+    <TestPlatformRoot Condition="$(TestPlatformRoot) == ''">..\..\..\</TestPlatformRoot>
+    <TestProject>true</TestProject>
+  </PropertyGroup>
+  <!-- Imports Common TestAssets props. -->
+  <Import Project="..\..\..\scripts\build\TestAssets.props" />
+
+  <PropertyGroup>
+    <AssemblyName>CUITTestProject</AssemblyName>
+    <TargetFrameworks>net451</TargetFrameworks>
+    <OutputType Condition="$(NetCoreAppTargetFrameWork) == 'true' ">Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="System" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Xml" />
+    <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework" />
+    <Reference Include="Microsoft.VisualStudio.QualityTools.CodedUITestFramework" />
+    <Reference Include="Microsoft.VisualStudio.TestTools.UITest.Common" />
+    <Reference Include="Microsoft.VisualStudio.TestTools.UITest.Extension" />
+    <Reference Include="Microsoft.VisualStudio.TestTools.UITesting" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.cuit">
+      <Version>15.6.0-preview-1329213</Version>
```

Comment:
Use `TestPlatformExternalsVersion`.

Reply:
I am not importing that props because that guy includes mstest v2 as well. my test needs mstest v1 reference